messenger/__main__.py

```diff
@@ -0,0 +1,41 @@
+import sys
+
+from argparse import ArgumentParser
+
+from .make import *
+
+def command_line():
+    """
+    Manages command line arguments.
+
+    Returns
+    -------
+    Namespace containing parsed arguments
+    """
+    parser = ArgumentParser(prog='messenger')
```

Comment:
Makes sense, should we rename the entire module to be build-messenger, or just this section of the code?